lib/foreman_tasks/dynflow/configuration.rb

```diff
@@ -1,146 +1,11 @@
-module ForemanTasks
-  class Dynflow::Configuration
-    # the number of threads in the pool handling the execution
-    attr_accessor :pool_size
-
-    # the size of db connection pool
-    attr_accessor :db_pool_size
-
-    # set true if the executor runs externally (by default true in procution, othewise false)
-    attr_accessor :remote
-    alias remote? remote
-
-    # what transaction adapater should be used, by default, it uses the ActiveRecord
-    # based adapter, expecting ActiveRecord is used as ORM in the application
-    attr_accessor :transaction_adapter
-
-    attr_accessor :eager_load_paths
-
-    attr_accessor :lazy_initialization
-
-    # what rake tasks should run their own executor, not depending on the external one
-    attr_accessor :rake_tasks_with_executor
-
-    # if true, the ForemanTasks::Concerns::ActionTriggering will make
-    # no effect. Useful for testing, where we mignt not want to execute
-    # the orchestration tied to the models.
-    attr_accessor :disable_active_record_actions
```

Comment:
When running the tests locally it fails with 
```ruby
foreman-tasks/test/foreman_tasks_test_helper.rb:11:in `<top (required)>': undefined method `disable_active_record_actions=' for #<ForemanTasks::Dynflow::Configuration:0x005580871810c0> (NoMethodError)
```

it needs to be added to the configuration in dynflow or foreman